sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+* Add support for Structured Outputs for Agents.
+* Add option to include file contents, when index search is used for Agents.
```

Comment:
File contents is included in the response? Will this be used e.g. when doing evaluation for grounding?

Reply:
Yes, evaluation can use this file as a context. Per my understanding user can add this parameter to `data_mapping` parameter of EvaluatorConfiguration.